recipy/log.py

```diff
@@ -169,7 +172,8 @@ def log_output(filename, source):
 
     db = open_or_create_db()
 
-    if option_set('data', 'file_diff_outputs') and os.path.isfile(filename):
+    if option_set('data', 'file_diff_outputs') and os.path.isfile(filename) \
+       and not is_binary(filename):
```

Comment:
Thanks, I didn't know about `all()`. 

Not sure what would be more idiomatic. I kind of prefer the `and`s, probably because not everybody knows `all()`...